src/main/java/org/team1540/robot2025/commands/AutoAlignCommands.java

```diff
@@ -40,34 +40,13 @@ public static Command alignToBranch(ReefBranch branch, Drivetrain drivetrain) {
     }
 
     public static Command alignToNearestBranch(Drivetrain drivetrain) {
-        return Commands.defer(
-                () -> {
-                    Pose2d closestBranch = new Pose2d();
-                    double closestDistance = Double.MAX_VALUE;
-
-                    for (Pose2d pose : FieldConstants.Reef.scorePositions) {
-                        pose = AllianceFlipUtil.maybeFlipPose(pose);
-                        double distance = RobotState.getInstance()
-                                .getEstimatedPose()
-                                .minus(pose)
-                                .getTranslation()
-                                .getNorm();
-                        if (distance < closestDistance) {
-                            closestDistance = distance;
-                            closestBranch = pose;
-                        }
-                    }
-
-                    Pose2d finalClosestBranch = closestBranch;
-                    return alignToPose(() -> finalClosestBranch, drivetrain);
-                },
-                Set.of(drivetrain));
+        return Commands.defer(() -> alignToPose(FieldConstants.closestBranch(), drivetrain), Set.of(drivetrain));
     }
 
     public static Command alignToNearestFace(Drivetrain drivetrain, BooleanSupplier isRight) {
         return Commands.defer(
                 () -> {
-                    Pose2d closestBranch = new Pose2d();
```

Comment:
shouldn't this be using the getClosestFace() functions you declared in FieldConstants?